stable/mongodb-replicaset/README.md

```diff
@@ -117,6 +117,18 @@ keys `user` and `password`, that for the key file must contain `key.txt`.  The u
 full `root` permissions but is restricted to the `admin` database for security purposes. It can be
 used to create additional users with more specific permissions.
 
+To connect to the mongo shell with authentication enabled, use a command similar to the following (substituting values as appropriate):
+
+```shell
+kubectl exec -it mongodb-replicaset-0 -- mongo mydb -u admin -p password --authenticationDatabase admin
+```
+
+To connect to the mongodb replica set via an application, use a mongo URI similar to:
+
+```
+mongodb://admin:password@mongodb-replicaset:27017/mydb?authSource=admin&replicaSet=rs0
```

Comment:
I would remove this. This may depend on the driver/language you use. Adding the credentials to the URL is usually not a good idea and I doubt it will work with spaces in the password.